Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings

```diff
@@ -1013,7 +995,7 @@
 "login.loginListSearchBarPlaceHolderTitle" = "Filtrele";
 
 /* The message displayed in alert when a user needs to set a passcode */
-"login.syncSetPasscodeAlertDescription" = "Senkronizasyon zincirini ayarlamak veya ayarları görmek için, öncelikle cihazınızda bir parola belirlemeniz gerekiyor.";
+"login.syncSetPasscodeAlertDescription" = "Senkronizasyon zincirine bir cihaz eklemek veya şifre senkronizasyonunu açmak veya kapatmak için, önce cihazınıza bir parola belirlemeniz gerekmektedir.";
```

Comment:
"Senkronizasyon zincirine bir cihaz eklemek veya şifre senkronizasyonunu açmak veya kapatmak için, önce cihazınıza bir pin kodu belirlemeniz gerekmektedir."